app/services/fetch_web_document.rb

```diff
@@ -28,7 +28,17 @@ def set_document_attachment
   end
 
   def fetch_file_content
-    content = HTTParty.get(@document.link)
+    begin
+      content = HTTParty.get(@document.link)
+    rescue StandardError => e
+      tempfile.write(e.message)
+      tempfile.rewind
+      set_document_attachment
+      tempfile.close
```

Comment:
We should probably not put the error in the document. I think setting `error_message` is the right thing to do.

Reply:
My mistake - the Job is doing this. See my other comment.